app/representers/policy_representer.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module PolicyRepresenter
+  include ThreeScale::JSONRepresenter
+
+  wraps_resource
+
+  property :id
```

Comment:
So we also will have an `id` in porta to identify them apart from [name][version]

Reply:
Yes that makes sense.
The update endpoint should either use the id either the name+version
The id is generic of REST endpoint, name+version is because it is handy